test/00_operators/OperatorTests.cu

```diff
@@ -2704,6 +2704,74 @@ TEST(OperatorTests, Cast)
   MATX_EXIT_HANDLER();
 }
 
+template<class TypeParam>
+TypeParam legendre_check(int n, TypeParam in) {
```

Comment:
Can you use `scipy.special.legendre` instead?

Reply:
don't think so.